sample-applications/blazing-coffee/BlazingCoffee/Client/Shared/CultureChooser.razor

```diff
@@ -53,11 +53,14 @@
     public async Task GetCulture()
     {
         var value = await LocalStorage.GetItemAsStringAsync("BlazorCulture");
-        if (string.IsNullOrEmpty(value))
+        var parsedValue = !string.IsNullOrEmpty(value) ? JsonSerializer.Deserialize<string>(value) : "en-US";
+
+        if (string.IsNullOrEmpty(parsedValue))
         {
-            value = "en-US";
+            parsedValue = "en-US";
```

Comment:
"en-US" can be extracted to a constant and reused as it is used in more than 1 place.